src/components/addPlayer.svelte

```diff
@@ -1,49 +1,34 @@
 <script>
-    import { count, players} from './stores.js';
-    
-    let name;
-    let bio;
-    let picture;
+  import {currPlayer, appendPlayers, resetPoints } from '../playerService.js';
```

Comment:
I've noticed that you use semicolons inconsistently, fwiw, the code loks cleaner without them but either way you'd want your codebase to be consistent, either with them or without.

Reply:
If there is no need for them I could live without them.